### ISSUE TITLE
Failover requests should take create a new request logger

### ISSUE THREAD

**Issue by rahulreddy** [scality/sproxydclient#99]

**Comment by DavidPineauScality** [comment]
I'm not exactly convinced by this: We're still processing the same request (and the request uids are here to help tracking how _one_ requests fares)
Generating a new ID may complexify things here, including for investigations.

Now, if you're talking about a sub-request id, that may be a possibility, but still generated quite a bit of complexity in understanding the logs.

**Comment by rahulreddy** [comment]
Yes, I meant the sub-request id since the request logger would be created using the req_id from S3.

I was helping @msegura debug an issue why certain requests were failing at sproxyd level and it was hard to track the individual failover requests at the nginx level as all the requests had the same "sub" request_id.
So instead of creating logger only once, for example right now at the beginning of a PUT operation, it could be moved over to`_failover` so that we get a different request_id combo for each failover request.
The following example is more easy to debug at nginx level

Failover#1 `a7ea4c3151407e76b10e:7a6691a0746cb6785797`
Failover#2 `a7ea4c3151407e76b10e:99bc9f79c649553d84c3`
Failover#3 `a7ea4c3151407e76b10e:4472d440c6546a7cdfd8`
Failover#4 `a7ea4c3151407e76b10e:d4e4f926ace52cf1651f`
Failover#5 `a7ea4c3151407e76b10e:1e62081125aaad441594 `

**Comment by DavidPineauScality** [comment]
I see, so it's more to be able to easily track at the next layer. I understand and support the request :)
